lib/web/assistant.go

```diff
@@ -189,6 +191,12 @@ type partialFinalizePayload struct {
 
 // runAssistant upgrades the HTTP connection to a websocket and starts a chat loop.
 func runAssistant(h *Handler, w http.ResponseWriter, r *http.Request, sctx *SessionContext) error {
+	// Initialize a tokenizer for prompt token accounting
+	tokenizerInstance, err := tokenizer.Get(tokenizer.Cl100kBase)
```

Comment:
nit: Can you please add to the comment that Cl100k tokenizer is being used by GPT-3 and GPT-4? Without context, it's not clear why we're using this particular one.